charts/tautulli/README.md

```diff
@@ -1,96 +1,15 @@
-# Tautulli
+# tautulli umbrella chart
```

Comment:
We should certainly identify this as an umbrella chart, but should still have some useful description about what it is and what it does in the README.

We should also have basic installation instructions (see deleted blocks below) for each of these charts in addition to a reference to the generic media-common that's being used behind the scenes.

A user of the chart will probably be more concerned about what the chart is, how to install it, and what options can be set.